command/root.go

```diff
@@ -97,7 +97,7 @@ var initContext = func() context.Context {
 // BasicClient returns an API client that borrows from but does not depend on
 // user configuration
 func BasicClient() (*api.Client, error) {
-	opts := []api.ClientOption{}
+	opts := make([]api.ClientOption, 0, 1)
```

Comment:
Is there a benefit to doing this over than just `make()` without numeric arguments?

Reply:
Not very much, just capacity is specified in advance.
https://play.golang.org/p/mRVgR_B6QNc

Reply:
Unless there is a strong reason to specify capacity 1, let's not pass any arguments for the sake of clarity.